flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/type/TypeConverters.java

```diff
@@ -238,4 +238,9 @@ public static TypeInformation createExternalTypeInfoFromInternalType(InternalTyp
 			throw new UnsupportedOperationException("Not support yet: " + type);
 		}
 	}
+
+	public static BaseRowTypeInfo toBaseRowTypeInfo(RowType t) {
```

Comment:
This util is a bit redundant. Direct new BaseRowTypeInfo(t.getFieldTypes(), t.getFieldNames())?

Reply:
done